src/layouts/hass-tabs-subpage.ts

```diff
@@ -228,6 +228,10 @@ class HassTabsSubpage extends LitElement {
         min-width: 0;
       }
 
+      :host(:not([narrow])) #toolbar-icon {
+        min-width: 40px;
```

Comment:
Doubt between this and 
```css
 position: absolute;
 right: 16px;
```
The `min-width` centers the tabs nice with the back button on the left, but only fixes this as long as we use only 1 icon. The `position: absolute` would fix it in all cases, but could overlap when there are multiple icons and tabs.